packages/client/hmi-client/src/page/project/components/code-editor.vue

```diff
@@ -501,13 +505,36 @@ async function createModelFromCode() {
 	flex-direction: column;
 	flex: 1;
 	padding-top: 1rem;
+	background-color: var(--surface-0);
 }
 
 .controls {
-	margin-left: 40px;
-	margin-bottom: 10px;
+	margin-left: 3rem;
+	margin-right: 1rem;
+	margin-bottom: 1rem;
 	display: flex;
 	gap: 10px;
+	justify-content: space-between;
+}
+
+.control-group {
+	display: flex;
+	gap: 1rem;
+}
+
+.outline-upload-button {
+	background-color: var(--surface) !important;
+	color: var(--text-color-primary) !important;
+	border: 1px solid var(--surface-border) !important;
+	width: 100%;
+	font-size: var(--font-body-small);
+}
+.outline-upload-button:hover {
+	background-color: var(--surface-hover) !important;
+}
```

Comment:
**No `!important` in this project**
* use `::v-deep(.outline-upload-button)`.
* Increase the specificity of your selector by doubling it `.outline-upload-button.outline-upload-button`
* or both `::v-deep(.outline-upload-button.outline-upload-button)`.